### PR TITLE
therock fetchLedger, fetchTransactions, fetchOrders, handleErrors

### PR DESCRIPTION
**Pull request by npomfret**
[therock] added fetchDeposits and fetchWithdrawals
[binance] just some docs links

### CODE REVIEW

File: js/therock.js [hunk 1]
```diff
@@ -351,6 +351,203 @@ module.exports = class therock extends Exchange {
...
 
+    async fetchWithdrawals (code = undefined, since = undefined, limit = undefined, params = {}) {
+        return this._fetchTransactions ('withdraw', code, since, limit, params);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        return this._fetchTransactions ('atm_payment', code, since, limit, params);
+    }
+
+    async _fetchTransactions (type, code = undefined, since = undefined, limit = undefined, params = {}) {
```
Comment (kroitor) [inline js/therock.js]: This is definitely a `fetchLedger`, not `fetchTransactions`, I will have to rework this.

Comment (npomfret) [reply js/therock.js]: I'd say that it could be fetchLedger, but for now it's fetchTransactions.

Comment (kroitor) [reply js/therock.js]: @npomfret plz check this out: https://github.com/ccxt/ccxt/pull/4179#discussion_r234361900

Comment (kroitor) [reply js/therock.js]: Also this is important: https://github.com/ccxt/ccxt/pull/4161#issuecomment-438710176

Comment (kroitor) [reply js/therock.js]: @npomfret i will add the edits to this PR to rewire the unified methods. If you can then take a look at the edits I made and reflect them in further PRs – that would really speed up the process  ) but if not, your help and effort is still very much appreciated ) thx )

**Pull request merged**